Index: src/i_Quiz33.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>class 고양이 {\r\n\r\n    int a;\r\n    int p;\r\n    String f;\r\n\r\n    int getA() {\r\n        return a;\r\n    }\r\n    void setA(int a) {\r\n        this.a = a;\r\n    }\r\n    int getP() {\r\n        return p;\r\n    }\r\n    void setP(int p) {\r\n        this.p = p;\r\n    }\r\n    String getF() {\r\n        return f;\r\n    }\r\n    void setF(String f) {\r\n        this.f = f;\r\n    }\r\n\r\n}\r\n\r\npublic class i_Quiz33 {\r\n    //고양이\r\n    public static void main(String[] args) {\r\n        고양이 cm = new 고양이();\r\n\r\n        cm.setA(13);\r\n        cm.setF(\"영희\");\r\n        cm.setP(3);\r\n\r\n        System.out.printf(\"길고양이가 몇마리있나요? %d / 몇분이서 관리하시나요? %d / 누가 밥을주시나요? %s\\n\", cm.getA(), cm.getP(), cm.getF());\r\n    }\r\n\r\n\r\n\r\n}\r\n
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/i_Quiz33.java b/src/i_Quiz33.java
--- a/src/i_Quiz33.java	(revision 8fcd8b269c15006c17323075a97ef18eb5962bc4)
+++ b/src/i_Quiz33.java	(date 1621280221575)
@@ -1,5 +1,4 @@
 class 고양이 {
-
     int a;
     int p;
     String f;
@@ -22,18 +21,14 @@
     void setF(String f) {
         this.f = f;
     }
-
 }
-
 public class i_Quiz33 {
     //고양이
     public static void main(String[] args) {
         고양이 cm = new 고양이();
-
         cm.setA(13);
         cm.setF("영희");
         cm.setP(3);
-
         System.out.printf("길고양이가 몇마리있나요? %d / 몇분이서 관리하시나요? %d / 누가 밥을주시나요? %s\n", cm.getA(), cm.getP(), cm.getF());
     }
 
